### PR TITLE
Cleanups and fixes

### PR DESCRIPTION
**Pull request by chanseokoh**
Just realized the assertion should have been `isGreaterThan` in #3176.

### CODE REVIEW

File: jib-maven-plugin/src/integration-test/java/com/google/cloud/tools/jib/maven/BuildImageMojoIntegrationTest.java [hunk 1]
```diff
@@ -560,7 +560,7 @@ public void testExecute_complex()
     Assert.assertNotEquals(digest, id);
     Assert.assertEquals(output, new Command("docker", "run", "--rm", id).run());
 
-    assertThat(getCreationTime(targetImage)).isAtLeast(before);
+    assertThat(getCreationTime(targetImage)).isGreaterThan(before);
```
Comment (mpeddada1) [inline jib-maven-plugin/src/integration-test/java/com/google/cloud/tools/jib/maven/BuildImageMojoIntegrationTest.java:563]: Good catch.

**Review by mpeddada1 — APPROVED** [review]

File: jib-gradle-plugin/src/integration-test/java/com/google/cloud/tools/jib/gradle/SingleProjectIntegrationTest.java [hunk 2]
```diff
@@ -372,7 +372,7 @@ public void testBuild_complex()
     Assert.assertNotEquals(digest, id);
     Assert.assertEquals(output, new Command("docker", "run", "--rm", id).run());
 
-    JibRunHelper.assertSimpleCreationTimeIsAfter(beforeBuild, targetImage);
+    assertThat(JibRunHelper.getCreationTime(targetImage)).isGreaterThan(beforeBuild);
     assertThat(getWorkingDirectory(targetImage)).isEqualTo("/");
```
Comment (mpeddada1) [inline jib-gradle-plugin/src/integration-test/java/com/google/cloud/tools/jib/gradle/SingleProjectIntegrationTest.java:376]: let's also convert the other assert statements in this test to Truth.

Comment (chanseokoh) [reply jib-gradle-plugin/src/integration-test/java/com/google/cloud/tools/jib/gradle/SingleProjectIntegrationTest.java:376]: I've been doing that, but there are too many. I think I'll just do it on the best effort basis. Don't want to spend too much time on low priority tasks for now.

File: jib-gradle-plugin/src/integration-test/java/com/google/cloud/tools/jib/gradle/JibRunHelper.java [hunk 3]
```diff
@@ -181,20 +183,10 @@ static void assertBuildSuccess(BuildResult buildResult, String taskName, String
...
-      throws IOException, InterruptedException {
+  static Instant getCreationTime(String imageReference) throws IOException, InterruptedException {
     String inspect =
         new Command("docker", "inspect", "-f", "{{.Created}}", imageReference).run().trim();
-    Instant parsed = Instant.parse(inspect);
-    Assert.assertTrue(parsed.isAfter(before));
+    return Instant.parse(inspect);
   }
 
   static void assertImageDigestAndId(Path projectRoot) throws IOException, DigestException {
```
Comment (mpeddada1) [inline jib-gradle-plugin/src/integration-test/java/com/google/cloud/tools/jib/gradle/JibRunHelper.java:192]: Let's make the name of this helper more descriptive?

File: jib-gradle-plugin/src/integration-test/java/com/google/cloud/tools/jib/gradle/SingleProjectIntegrationTest.java (hunk 2 above)
Comment (mpeddada1) [reply jib-gradle-plugin/src/integration-test/java/com/google/cloud/tools/jib/gradle/SingleProjectIntegrationTest.java:376]: Makes sense. Having a mix of Truth and Junit is usually okay (neither discouraged nor encouraged) - we can just migrate in a separate PR when possible.

File: jib-gradle-plugin/src/integration-test/java/com/google/cloud/tools/jib/gradle/JibRunHelper.java [hunk 4]
```diff
@@ -189,7 +190,8 @@ static Instant getCreationTime(String imageReference) throws IOException, Interr
     return Instant.parse(inspect);
   }
 
-  static void assertImageDigestAndId(Path projectRoot) throws IOException, DigestException {
+  static void assertValidImageDigestAndIdOutputFiles(Path projectRoot)
```
Comment (mpeddada1) [inline jib-gradle-plugin/src/integration-test/java/com/google/cloud/tools/jib/gradle/JibRunHelper.java:193]: hm how about `assertThatExpectedImageDigestAndIdReturned`?

**Review by mpeddada1 — APPROVED** [review]

File: jib-gradle-plugin/src/integration-test/java/com/google/cloud/tools/jib/gradle/JibRunHelper.java (hunk 4 above)
Comment (chanseokoh) [reply jib-gradle-plugin/src/integration-test/java/com/google/cloud/tools/jib/gradle/JibRunHelper.java:193]: Fixed in #3179.

**Pull request merged**